customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -60,7 +60,7 @@ function run_ib_bw_gdr(){
   dbg "ib_write_lb_${device}: $ib_bandwidth Gbps"
   if (( $(echo "$ib_bandwidth < $EXP_IB_BW" | bc -l) )); then
 	 log "$IB_WRITE_BW_OUT2"
-    die 1 "$FUNCNAME: $IB_WRITE_BW, IB=$device, $device_peer, IB BW (expected > $EXP_IB_BW Gbps, but measured $ib_bandwidth Gbps"
+    die 1 "$FUNCNAME: $IB_WRITE_BW, IB=$device, $device_peer, IB BW (expected > $EXP_IB_BW Gbps, but measured $ib_bandwidth Gbps. FaultCode: NHCNA"
```

Comment:
this is an actual failure of IB BW

Reply:
I will change it to the IBPerformance fault code